src/components/StatusBar/index.tsx

```diff
@@ -58,6 +80,16 @@ const StatusBar = () => {
                 </span>
             </Tooltip>
 
+            <StatusBarToggleButton
+                data-action-name={ACTION_NAME.TOGGLE_PRETTIFY}
+                isActive={!isPrettified}
+                offIcon={<AutoFixOffRoundedIcon/>}
+                tooltipTitle={isPrettified ?? false ?
+                    "Prettify Off" :
+                    "Prettify On"}
+                onClick={handleStatusButtonClick}
+                onIcon={<AutoFixHighRoundedIcon/>}/>
+
             <LogLevelSelect/>
```

Comment:
how about
```suggestion
            <LogLevelSelect/>

            <StatusBarToggleButton
                data-action-name={ACTION_NAME.TOGGLE_PRETTIFY}
                disabled={isPrettifyButtonDisabled}
                isActive={isPrettified ?? false}
                icons={{
                    active: <AutoFixHighRoundedIcon/>,
                    inactive: <AutoFixOffRoundedIcon/>,
                }}
                tooltipTitle={(isPrettified ?? false) ?
                    "Turn off Prettify" :
                    "Turn on Prettify"}
                onClick={handleStatusButtonClick}/>
```
1. I feel it's better to put the toggle button towards the corner of the status bar. Feel free to object.
2. Added the `disabled` property.
3. Added the `icons` property. (Please see the updated interface in `<StatusBarToggleButton/>`.
4. Updated `tooltipTitle` text to hopefully make things more clear to users.
5. Corrected `isActive` (or maybe i understood your proposed behaviour wrong. feel free to object.)

Reply:
Fixed! Previously, I thought the icon shows the action that the user wanted to do, for example, when it is inactive and user want to click the button, then the user must want to turn on the prettifying, so the icon is the "On" state. But I think yours make sense, because this is in status bar, so the icon should reflect the current status.